lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -77,7 +77,14 @@ Function Chocolatey-Install-Upgrade
             Fail-Json $result "Chocolatey bootstrap installation failed."
         }
         $result.changed = $true
-        $script:executable = "C:\ProgramData\chocolatey\bin\choco.exe"
+        if (Test-Path "Env:\ChocolateyInstall")
```

Comment:
We prefer explicit option names somehow, something like:
```powershell
if (Test-Path -Path "Env:\ChocolateyInstall")
```
Although it's not been enforced at the moment as we're waiting for the Testing team to implement this.